OpenRA.Mods.Common/Traits/Render/WithRangeCircle.cs

```diff
@@ -28,6 +28,15 @@ class WithRangeCircleInfo : ConditionalTraitInfo, IPlaceBuildingDecorationInfo
 		[Desc("Color of the circle")]
 		public readonly Color Color = Color.FromArgb(128, Color.White);
 
+		[Desc("Contrast width")]
+		public readonly float Width = 1;
+
+		[Desc("Color of the border of the circle.")]
+		public readonly Color BorderColor = Color.FromArgb(96, Color.Black);
```

Comment:
The different traits are swapping between `Contrast` and `Border`. Can we please use `Border` everywhere?

Reply:
You mean contrast?

Reply:
No, I mean Border because this defines the colour of the border. The border color may have strong contrast against the main colour, but this is not required.

Reply:
Then you are commenting the wrong line as it is meant to stay then.